docs/docs/monitoring/analytics.mdx

```diff
@@ -76,11 +76,25 @@ will be disabled.
 
 TODO: general instructions.
 
-TODO: instructions for AWS RDS
+You can create a PostgreSQL instance using Amazon Relational Database Service (RDS).
+First you must set up Amazon RDS by completing the instructions listed [here](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_SettingUp.html).
+Next, create a PostgreSQL DB instance which is the environment that will run your PostgreSQL database to which the analytics pipeline will stream the transformed conversational data.
+The instructions you must follow are listed in the **Creating a PostgreSQL DB instance** [section](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_GettingStarted.CreatingConnecting.PostgreSQL.html#CHAP_GettingStarted.Creating.PostgreSQL)
+
+Finally, when running the Analytics pipeline Docker container, set the environment variable `RASA_ANALYTICS_DB_URL` to the PostgreSQL Amazon RDS DB instance URL
```

Comment:
do we want to link to https://11653--rasa-community-docs.netlify.app/docs/rasa/deploy/deploy-rasa-pro-services#docker-container-configuration-reference ?